code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -413,6 +414,53 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 /obj/item/stack/tile/brass/fifty
 	amount = 50
 
+/*
+ * Bronze
+ */
+
+GLOBAL_LIST_INIT(bronze_recipes, list ( \
+	new/datum/stack_recipe("wall gear", /obj/structure/girder/bronze, 2, time = 20, one_per_turf = TRUE, on_floor = TRUE), \
+	null,
+	new/datum/stack_recipe("bronze hat", /obj/item/clothing/head/bronze), \
+	new/datum/stack_recipe("bronze suit", /obj/item/clothing/suit/bronze), \
+	new/datum/stack_recipe("bronze boots", /obj/item/clothing/shoes/bronze), \
+	null,
+	new/datum/stack_recipe("bronze chair", /obj/structure/chair/bronze, 1, time = 0, one_per_turf = TRUE, on_floor = TRUE), \
+))
+
+/obj/item/stack/tile/bronze
+	name = "brass"
+	desc = "On closer inspection, what appears to be wholly-unsuitable-for-building brass is actually more structurally stable bronze."
+	singular_name = "bronze sheet"
+	icon_state = "sheet-brass"
+	item_state = "sheet-brass"
+	icon = 'icons/obj/stack_objects.dmi'
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	throwforce = 10
+	max_amount = 50
+	throw_speed = 1
+	throw_range = 3
+	turf_type = /turf/open/floor/bronze
+	novariants = FALSE
+	grind_results = list("iron" = 5, "copper" = 3) //we have no "tin" reagent so this is the closest thing
+
+/obj/item/stack/tile/bronze/attack_self(mob/living/user)
+	if(is_servant_of_ratvar(user)) //still lets them build with it, just gives a message
+		to_chat(user, "<span class='danger'>Wha... what is this cheap imitation crap? This isn't brass at all!</span>")
+	..()
+
+/obj/item/stack/tile/bronze/Initialize(mapload, new_amount, merge = TRUE)
+	recipes = GLOB.bronze_recipes
+	. = ..()
+	pixel_x = 0
```

Comment:
what's setting these off 0?

Reply:
I'm unsure. It was in the brass sheet's Initialize, so I moved it here since they shared the same sprite.